multi-market-integrations.md

```diff
@@ -0,0 +1,88 @@
+## **Organization Key (`organizationKey`): Multi-Market and Region Isolation**
+
+The Organization Key (`organizationKey`) is a globally unique identeifier (GUID).
+It defines which market or country your data belongs to and guarantees that all IDs are correctly scoped and isolated across regions. 
+
+
+### **How to implement a Multi-Market or Multi-Region Integration**
+
+---
+
+### **1. Use the `globalOrganization` query**
+
+Always start by calling the [`globalOrganization`](https://api.connectedcars.io/graphql/graphiql/) query.
+
+* This query does **not require authentication** — you can always get the complete list of organizations and their configuration.
+* It works globally, so any API endpoint returns the same results.
+* Each `GlobalOrganization` includes a unique `key` — this is the `organizationKey` you will use to scope all IDs.
+* The query also returns important domain information for each organization.
+
+The most relevant domains are:
+
+* **`apiDomain`** — the GraphQL API endpoint for that organization.
+  [See API documentation](https://docs.connectedcars.io/#/?id=graphql-api)
+
+* **`authDomain`** — the authorization endpoint for service account tokens and logins.
+  [See Auth documentation](https://docs.connectedcars.io/#/?id=auth-api)
+
+Other domains may be relevant depending on your use case.
+
+---
+
+### **2. Service account mapping and authentication**
+
+Each organization must have its own service account.
```

Comment:
Technically service accounts are tied to namespaces. In most cases the would be `${organizationKey}:workshop` but in some cases it might be `leasing`, `default` or something else. So in this case I guess they should either map the `organizationKey` to both namespace and service account or fetch the relevant namespace from the `GlobalOrganization.namespaces` based on the product.